doc/user/content/installation/install-on-gcp/_index.md

```diff
@@ -190,6 +190,9 @@ If you want to use `jq` and do not have `jq` installed, install.
 
 {{< /warning >}}
 
+{{< tabs >}}
```

Comment:
https://preview.materialize.com/materialize/32299/self-managed/v25.1/installation/install-on-gcp/#b-set-up-gcp-kubernetes-environment-and-install-materialize